tests/Makefile.am

```diff
@@ -718,6 +718,17 @@ TESTS +=  \
 endif
 endif
 
+if ENABLE_OMHTTP
+TESTS +=  \
```

Comment:
it would be good to have test with valgrind support as well. They are easy to do, all three-liners:
```
#!/bin/bash
export USE_VALGRIND="YES"
source ${srcdir:=.}/non-valgrind-test.sh
```
Test files end in `-vg.sh`

Reply:
Woops, overlooked this. I'll add a few valgrind tests.